provider/lpwindow/compute_task.go

```diff
@@ -150,7 +150,7 @@ func (t *WdPostTask) Do(taskID harmonytask.TaskID, stillOwned func() bool) (done
 		return false, err
 	}
 
-	postOut, err := t.doPartition(context.Background(), ts, maddr, deadline, partIdx)
+	postOut, err := t.DoPartition(context.Background(), ts, maddr, deadline, partIdx)
```

Comment:
So below here we need to capture no_send and add it to wdpost_proofs, which will later be picked up by the submit task - which will have a different taskID than the compute task.

Alternatively we can just put a no_send on wdpost_proofs, and simply never pick that up in the submit task.

Not blocking the merge on this, but should be fixed, otherwise we'll do the send in test mode.